frontend/src/app/Settings/Facility/Components/ManageDevices.tsx

```diff
@@ -31,7 +32,7 @@ const ManageDevices: React.FC<Props> = ({
   return (
     <div className="prime-container card-container">
       <div className="usa-card__header">
-        <h2 className="font-heading-lg">Manage devices</h2>
+        <h2 className="font-heading-lg">{editDevicePageTitle}</h2>
```

Comment:
this is the title for selecting devices for a facility in the manage facility page - I don't think swapping this for the new copy makes sense here